packages/core/src/Card/stories/Variants.tsx

```diff
@@ -71,7 +71,7 @@ export const Variants = () => {
         </HvCardContent>
       </HvCard>
 
-      <HvCard className={classes.card} bgcolor="atmo1">
+      <HvCard className={classes.card} bgcolor="atmo1" statusColor="negative">
```

Comment:
As we're giving it a semantic color, we could also give it a semantic icon (`HvCard`'s `icon` prop) in a couple of cards

Reply:
Yes, I added this.